src/openrct2/ride/RideRatings.cpp

```diff
@@ -454,15 +454,18 @@ static void ride_ratings_score_close_proximity_in_direction(TileElement* inputTi
                 }
                 break;
             case TILE_ELEMENT_TYPE_PATH:
-                if (abs((int32_t)inputTileElement->base_height - (int32_t)tileElement->base_height) <= 2)
+                if (abs(static_cast<int32_t>(inputTileElement->base_height) - static_cast<int32_t>(tileElement->base_height))
+                    <= 2)
                 {
                     proximity_score_increment(PROXIMITY_PATH_SIDE_CLOSE);
                 }
                 break;
             case TILE_ELEMENT_TYPE_TRACK:
                 if (inputTileElement->AsTrack()->GetRideIndex() != tileElement->AsTrack()->GetRideIndex())
                 {
-                    if (abs((int32_t)inputTileElement->base_height - (int32_t)tileElement->base_height) <= 2)
+                    if (abs(static_cast<int32_t>(inputTileElement->base_height)
+                            - static_cast<int32_t>(tileElement->base_height))
+                        <= 2)
```

Comment:
Once this is replaced with `GetBaseZ()`, we can get rid of the casts. (Note for later)

Reply:
Noted, will followup